xml/System.Net.Quic/QuicAbortDirection.xml

```diff
@@ -40,7 +40,7 @@
       </ReturnValue>
       <MemberValue>3</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Abort both sides of the stream, that is, <see cref="F:System.Net.Quic.QuicAbortDirection.Read" /> and <see cref="F:System.Net.Quic.QuicAbortDirection.Write" />) at the same time.</summary>
```

Comment:
Missing opening parenthesis, Not sure what the punctuation rules are when combined with parentheses so feel free to edit further

```suggestion
        <summary>Abort both sides of the stream (i.e. <see cref="F:System.Net.Quic.QuicAbortDirection.Read" /> and <see cref="F:System.Net.Quic.QuicAbortDirection.Write" />) at the same time.</summary>
```

Reply:
```suggestion
        <summary>Abort both sides of the stream, that is, <see cref="F:System.Net.Quic.QuicAbortDirection.Read" /> and <see cref="F:System.Net.Quic.QuicAbortDirection.Write" />, at the same time.</summary>
```

Reply:
We are not supposed to use abbreviations like i.e. and etc. in the docs.